content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -233,15 +233,9 @@ can happen, according to:
 
 ## Pod disruption conditions {#pod-disruption-conditions}
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state for_k8s_version="v1.31" state="stable" >}}
```

Comment:
Improvement for 'feature-state' shortcode usage here, consider using the new feature_gate_name option, making the feature state more data-driven.

```suggestion
{{< feature-state feature_gate_name="PodDisruptionConditions" >}}
```